tests/unit/options/unit_tests_sender_option_parser.rb

```diff
@@ -358,6 +358,17 @@ def test_sender_option_parser_user_msg_content_from_file_long
     )
   end # test_sender_option_parser_user_msg_content_from_file_long
 
```

Comment:
Should contain test of default value for --content-type.

Reply:
that is tested by the tests for -L, M, --msg-content tests, there --content-type assumes default value, because it is not being set

or do you mean that I should check that options.content_type == Defaults::DEFAULT_CONTENT_TYPE?

Reply:
you do, all right, it is obvious from the other tests in the file.

done

Reply:
ok